source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,150 @@
+   
+Teams that are not open for anyone to join are indicated by:
+ 
+.. image:: ../images/private_team.png 
+ 
+Public channels are indicated by: 
+ 
+.. image:: ../images/public_channel.png
+
+Private channels are indicated by:
+ 
+.. image:: ../images/private_channel.png  
+
+When a team is added, the ```Town Square```, ```Off-Topic``` channels will also be added as default, as well as any default channels set in the `ExperimentalDefaultChannels config setting <https://docs.mattermost.com/administration/config-settings.html?highlight=configuration%20settings#default-channels-experimental>`_. 
+
+When a channel is added without setting the team explicitly, the team will be shown in the Team and Channel Membership listing, but it will not be added to the group specifically. Because of this dependency, when the channel is removed, the team will also be removed.  Teams are listed in parentheses after the channel name in the channel selector.
+
+Teams and channels membership synchronization
+----------------------------------------------
+
+For new users, default teams and channels will be added when they log in for the first time. For existing users, default teams and channels will be added to after the next scheduled AD/LDAP sychronization. 
+
+It may take a few seconds to load all team and channel memberships for a user depending on the number of teams and channels the group is defaulted to. In our testing, it took 6 seconds for an organization with 200,000 users and 30,000 linked groups.
+
+.. note::
+Users are not removed from the team or channel on subsequent synchronizations of the AD/LDAP groups.  Users will need to be manually removed from the team or channel per the existing functionality. This is being considered for a future improvement. They will not be re-added if they were manually removed or removed themselves.
+
+.. image:: ../images/Team_Channel_Membership_Sync.png
+
+Remove configured teams and channels from a group
+-------------------------------------------------
+To remove a team or channel configured for a group, click “Remove” to the right of the team or channel name. Users already part of the team and channel will not be removed from that channel by this action. 
+
+View users belonging to the group
+---------------------------------
+
+Users who have logged in and accessed Mattermost will be visible in the members list on the group object. Members are read-only at this time and new members can be added through management in your AD/LDAP system. 
+
+.. image:: ../images/Group_Members.png
+
+Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels. If a user is removed from an AD/LDAP group and then later re-added, they will be defaulted again into the teams and channels configured in the group. If a user is deactivated in AD/LDAP, they be removed from the group and will lose access to the teams and channels configured.
```

Comment:
maybe put "lose access to Mattermost"? (If they're deactivated and reactivated, they may still maintain access to the channels -- not 100% sure, should maybe double check)

Reply:
Linda is verifying in her testing.  Can update this section with that information as soon as I have it.  Otherwise, added "lose access to Mattermost".